src/main/java/tatracker/logic/commands/session/FilterSessionCommand.java

```diff
@@ -43,11 +43,21 @@ public FilterSessionCommand(SessionPredicate predicate) {
     @Override
     public CommandResult execute(Model model) {
         requireNonNull(model);
-        model.updateFilteredSessionList(predicate);
-        String keywords = predicate.getKeywords();
-        return new CommandResult(
-                String.format(MESSAGE_SUCCESS, keywords),
-                Action.FILTER_SESSION);
+        String returnMsg = "";
+
+        String sessionType = predicate.getSessionType().isPresent() ? predicate.getSessionType().get().toString() : "";
+        String date = predicate.getDate().isPresent() ? predicate.getDate().get().toString() : "";
+        String moduleCode = predicate.getModuleCode().isPresent() ? predicate.getModuleCode().get() : "";
```

Comment:
instead of isPresent

.map([Class name]::toString).orElse("")

We should avoid ?: where possible